modules/test_extension_commands.py

```diff
@@ -52,17 +51,8 @@ def _on_done(workspace_edit: WorkspaceEdit):
                 print(workspace_edit)
                 return
 
-            parsed_worspace_edit = parse_workspace_edit(workspace_edit)
-
-            def open_changed_file(result):
-                window = self.view.window()
-                if window and parsed_worspace_edit:
-                    for uri in parsed_worspace_edit:
-                        open_and_focus_uri(window, uri)
```

Comment:
Without this code, the test class isn't focused anymore.

Reply:
I don't have time to look into it now but do you have some simple way of reproducing (triggering this code)?

Reply:
I would like to see exactly what is happening here since it feels redundant on the first sight.

Reply:
The difference is that this code does also focus the changed views.

See also https://github.com/sublimelsp/LSP/pull/2079#pullrequestreview-1135237911